dist/meteo-france-weather-card.js

```diff
@@ -99,15 +99,18 @@ function getVigilance(color, alertEntity) {
 
 function getRainForecast(rainForecastEntity) {
   
-  let rainForecastColors = ["rgba(95,155,234,0.1)", "rgba(95,155,234,0.4)", "rgba(95,155,234,0.7)", "rgba(95,155,234,1)"];
-  let rainForecastTexts = ["Pas de pluie", "Pluie faible", "Pluie modérée", "Pluie forte"];
-  
+  let rainForecastColors = new Map([
+	    ['Temps sec', "0.1"],
+	    ['Pluie faible', "0.4"],
+	    ['Pluie modérée', "0.7"],
+	    ['Pluie forte',    "1"]
```

Comment:
```suggestion
	    ['Temps sec', 0.1],
	    ['Pluie faible', 0.4],
	    ['Pluie modérée', 0.7],
	    ['Pluie forte', 1]
```
This should work, otherwise keep the string